Eigen/src/Core/MathFunctions.h

```diff
@@ -435,7 +435,12 @@ struct round_retval
   struct arg_impl {
     static inline Scalar run(const Scalar& x)
     {
+      #if defined(EIGEN_HIP_DEVICE_COMPILE)
+      // HIP does not seem to have a native device side implementation for the math routine "arg"
+      using std::arg;
+      #else 		  
```

Comment:
This isn't really a review question, more curiosity.  @whchung should HIP have an implementation for std::arg?
"Calculates the phase angle (in radians) of the complex number."